modules/ROOT/partials/configuration/markdown.adoc

```diff
@@ -0,0 +1,69 @@
+[[configuration-option]]
```

Comment:
@HAFRMO I have added this outline template for `markdown`, but at this stage im not sure we have any config options for customer facing docs right?

Reply:
We do have the following options:
`markdown_symbols`, an object consisting of key/value pairs, where the key is the text ( For example, `C` ) and the output ( for example, `©` ). These are symbols, and are translated when surrounded by parenthesis ( For example, `(C)` is converted to `©`, not just `C` ).
This currently defaults to the following, though it may be expanded in the future:
```
{
  C: '©',
  TM: '™',
  R: '®'
}
```

Additionally it also uses the plugin emotions and it's emojis. As these can be image files, this can be more than just UTF symbols. These emojis are surrounded by `:` instead of parenthesis. So `:smile:` will be translated for these, not `(smile)` ( though, yes, this means that you can have both `:smile:` and `(smile)` and convert them both to different ( or same! ) output. Fun! )
For this to work the emoticon plugin needs to be enabled and set up properly. No additional options needed to make this work.